fimsauthz-api/utm-roles.yml

```diff
@@ -0,0 +1,131 @@
+    examples:
+        - "An IT admin manages FIMS-AZ logfiles."
+        - "An IT admin reads the master USS list to update FIMS-AZ's data table containing the USSs, their scopes and their credentials."
+
+FIMS_SME:
+    description: |
+        A role is assigned to user who is a FIMS Subject Matter Expert (SME) who
+        manages airspace constraints and other aeronautical data
+        This user could be a client to the FIMS Ops server using a User Interface (rather than CLI).
+        This role is typically associated with a user whose authorization is out of band WRT FIMS oauth.
+    identity_owner: ANSP
+    examples:
+        - "An airspace Subject Matter Expert composes an airspace constraint and submits this constraint to FIMS_OPS."
+
+FIMS_AUTHZ:
+    description: This is the service role supporting the FIMS Authorization server.
+    This role cannot create, modify or delete UTM Identity data, rather access is read-only.
+    identity_owner: UTM_AUTHORITY
+    examples:
+        - "When FIMS AZ services the token endpoint, it reads a data table containing USS identities and their scopes."
+
+FIMS_OPS:
+    description: This role manages the interface between the USS Network and
+    the ANSP.  This role is assigned to the FIMS_OPS server.
+    identity_owner: UTM_AUTHORITY
+    examples:
+        - "FIMS_OPS is authorized to inject an airspace constraint to the USS Network."
+    scopes:
+        - utm.nasa.gov_write.constraint
+
+VEHICLE_AUTHORITY:
+    identity_owner: TBD; May be a set of trusted commercial entities
+    description: |
+      This role allows creation, modifcation and deletion of vehicle registrations.
+      This role is typically associated with a user whose authentication and
+      authorization is out of band WRT FIMS oauth.
+      These users are small set of trusted vehicle subject matter experts who manage vehicle registrations.
+    examples:
+       - "A vehicle subject matter expert adds a new vehicle UVIN."
+
+VEHICLE_REGISTRATION_CHECKER:
+    description: |
+      This is the service role supporting vehicle registration data providers.
+      This role cannot create, modify or delete vehicle registrations.
+    identity_owner: VEHICLE_AUTHORITY
+    examples:
+      - "A vehicle registration data provider protects its endpoints with UTM uvin scopes."
+      - "USS queries a vehicle data provider using its uvin scopes to check whethter a UVIN is registered."
+      - "A Public Safety USS queries a vehicle data provider using its uvin scopes to learn the vehicle's owner."
+
+STRATEGIC_CM:
+    description: |
+       This is the service role supporting USS Strategic Confict Management data providers.
+       These data providers can be pull-based such as Grid Cell data, or a push-and-pull based
+       such as NASA's Discovery Service.
+    identity_owner: USS Network
+    examples:
+       - "NASA's USS Discovery Service PUTs USSInstance data to USS endpoints protected by write.stategic_cm scope."
```

Comment:
update with new scope name, if we are changing.

Reply:
agreed